book/src/porting/choosing_a_compiler.md

```diff
@@ -1,6 +1,6 @@
 # Choosing a compiler
 
-To compile [`mlibc`](https://github.com/managarm/mlibc) and any userspace programs which link against it, you'll need a suitable compiler. For that, you will need to build a full [OS Specific Toolchain](https://wiki.osdev.org/OS_Specific_Toolchain). Using your host's compiler or a generic toolchain could lead into strange build failures and is not recommended nor supported.
```

Comment:
I am not convinced that using a fork of the osdev.org wiki article should be preferred here, given that the OG wiki is active and publically editable.